source/crud/tests/unified/updateOne-comment.yml

```diff
@@ -29,7 +29,7 @@ tests:
         object: *collection0
         arguments:
           filter: &filter { _id: 1 }
-          update: &update { x: 22 }
+          update: &update { $set: {x: 22} }
```

Comment:
@comandeo: Another case where Ruby might not be validating its arguments for `updateOne` (and possibly `updateMany` and `replaceOne`).

Reply:
Noted.